CMakeLists.txt

```diff
@@ -6,6 +6,11 @@ cmake_minimum_required(VERSION 3.22)
 # src/CMakeLists.template.cmake
 cmake_policy(VERSION 3.22...3.31)
 
+# We aren't using C++20 modules, so disable scanning for them to avoid
+# clang-scan-deps-notfound errors.
+# See also https://discourse.cmake.org/t/cmake-3-28-cmake-cxx-compiler-clang-scan-deps-notfound-not-found/9244/3
+set(CMAKE_CXX_SCAN_FOR_MODULES OFF)
```

Comment:
Do we need this and the change below? Last time I tried, I didn't change anything, but I see that the clang-scan-deps issues is resolved now.